policyengine_uk/tests/policy/baseline/gov/hmrc/tax_free_childcare/tax_free_childcare_income_condition.yaml

```
@@ -0,0 +1,91 @@
+# Tests for age under 18 bracket (threshold: £1,331 quarterly)
+- name: Under threshold for age 15 with low income and no other income sources - ineligible
+  period: 2025
+  input:
+    age: 15
+    total_income: 5000  
```

Use the comma separator throughout
```suggestion
    total_income: 5_000
```